extensions/twitch/src/following.tsx

```diff
@@ -67,14 +61,11 @@ export default function main() {
           <ActionPanel>
```

Comment:
This action appears almost everywhere! You can create utils.tsx and link them here